data/blog/bitcoin-grants-6th-wave.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: "Sixth Wave of Bitcoin Grants"
+date: '2024-07-22'
+tags: ['OpenSats', 'grants', 'bitcoin']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/TODO.jpg']
+summary: "OpenSats is funding 8 more open-source projects in the bitcoin ecosystem."
+---
+
+We are excited to announce the sixth wave of Bitcoin grants, reaffirming our commitment to fostering innovation and development within the open-source community. 
+An important part of our mission is to provide essential funding to developers and projects that advance the Bitcoin ecosystem.
+
+In this latest round, we are proud to support eight remarkable projects that exemplify the spirit of open-source and technological advancement. In no particular order, these projects are:
+
+-   [Bitcoin Jungle](#bitcoin-jungle)
```

Comment:
I'd suggest we move Bitcoin Jungle to the end of the list. We usually lead with the tech, and end on education.

Reply:
Done 🖖